mne_bids/tests/test_mne_bids.py

```diff
@@ -45,8 +46,29 @@
     shell = True
 
 
-# MEG Tests
-# ---------
+def test_channels_tsv_units():
+    """Test that the original units are assigned if available."""
+    output_path = _TempDir()
+
+    # We use a brainvision testing file to check that raw._orig_units
+    # are assigned as appropriate. First write a channels.tsv
+    data_path = op.join(base_path, 'brainvision', 'tests', 'data')
+    raw_fname = op.join(data_path, 'test.vhdr')
+    raw = mne.io.read_raw_brainvision(raw_fname)
+    channels_tsv_name = op.join(output_path, 'channels.tsv')
+    _channels_tsv(raw, channels_tsv_name)
```

Comment:
in general, we don't test private functions. Also since we already have brainvision tests, couldn't we read the files written in that test and check?

Reply:
yes, I'll move it there